geoips/plugins/yaml/workflows/abi.yaml

```diff
@@ -0,0 +1,155 @@
+interface: workflows
+family: order_based
+name: abi_infrared
+docstring: |
+  ABI Infrared Workflow.
+spec:
+  steps:
+    - reader:
+        name: abi_netcdf
+        arguments:
+          variables: ["B14BT"]
```

Comment:
Will this YAML eventually reference all the "product" workflow YAMLs to produce duplicate final output to the existing "run_procflow" commands?  So this is the start of the "final" workflows that will produce what we are currently producing from our test scripts ?

Reply:
Essentially, yes, that's right.